tests/drivers/spi/spi_loopback/boards/nucleo_g474re.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics
```

Comment:
Please use your copyright

Reply:
Those where copied files, from other examples, so not really my copyright. (and to be honest I don't care to much about my copyright for FOSS projects anyway)

Reply:
I get your point but my reasoning is the following:
Even in case of copy/paste I consider that there is an effort of doing the effort and testing.
Besides the version of the files you copy might be different of the original version of the file that was initially "copyrighted" 
Doing otherwise, we'd end up having copyright info that totally offset vs reality, and these could be used for various stats, which are valuable to the project, when they are reflect reality.

But ok, I won't block for this.

Reply:
I have no problem changing it, but I just feel weird when I copy a file and remove 1 or 2 lines and than put my copyright in it.  On the other hand that is mostly with very trivial files I doubt the original copyright holder would have a problem with it, in the end the license is what is most important.  

But is there any "global" Zephyr guidance for cases like this ? 

Let me put back the "Richard Osterloh <richard.osterloh@gmail.com>" copyright because that file was already there and put my copyright in the new files (even if they are copied, but I changed them)